dev/Common/ColorConversion.cpp

```diff
@@ -14,12 +14,12 @@ Hsv::Hsv(double h, double s, double v) : h{ h }, s{ s }, v{ v }
 {
 }
 
-bool TryParseInt(const wstring_view& s, _Out_ unsigned long *outParam)
+bool TryParseInt(const wstring_view& s, unsigned long *outParam)
 {
```

Comment:
Let's leave it here -- by default I think pointer params are _In_ so this might cause /analyze failures. That said, we should rewrite these functions to return std::optional<unsigned long> and not have an out param. You want to maybe do that cleanup here?